### DISCUSSION TITLE
How can I select all nets besides clock, VSS and VDD?

### DISCUSSION THREAD

**Discussion by oharboe** [The-OpenROAD-Project/OpenROAD#3416] [General]
I'm interested in selecting all nets in `make gui_place` to see if there are any pathological routes. A pathological route in my case can be if buffers are placed in the wrong location.  I have vertical/horizontal routes and a buffer must be placed between the macros, not outside of them. Example from mock-array-big below.

If I select "*", I get a solid yellow...

![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/2798822/c7648a88-4509-4514-ad56-581cc41df938)

![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/2798822/fe5ffbab-125b-4609-bebf-123140995bdd)

**Comment by maliberty** [comment]
You can use Windows -> Selected/Highlighted Shapes and in the window you can deselect VDD & VSS.

**Reply by oharboe** [reply]
So here I have all nets, except VDD & VSS.

The "ray" pattern is the clock. My expectation is that this should saily through detailed routing easily...

![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/2798822/035c67c2-ddf1-4a1f-bdf6-d5b2ec4cf1e2)

**Reply by oharboe** [reply]
Same with `make gui_cts`:

![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/2798822/603baa6d-49b6-40a7-a49e-95f00108bb6c)
